src/imio/esign/tests/test_utils.py

```
@@ -450,6 +450,30 @@ def test_add_files_with_duplicate_filenames(self):
+
+        annex_uid = self.uids[0]
+        annex = api.content.get(UID=annex_uid)
+
+        sid, session = add_files_to_session(signers, (annex_uid,))
+        self.assertEqual(len(session["files"]), 1)
+        self.assertEqual(session["files"][0]["filename"], "annex0.pdf")
+        self.assertEqual(session["files"][0]["title"], "Annex 0")
+        # edit annex and add again, still one annex in session and data are updated
+        annex.file.filename = u"new_filename.pdf"
+        annex.setTitle('New title')
+        sid, session = add_files_to_session(signers, (annex_uid,))
```

Ruff (0.15.4)

[warning] Unpacked variable `sid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)